certmanager/generate_cert_test.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package certmanager
+
+import (
+	"encoding/pem"
+	"crypto/rsa"
+	"crypto/x509"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+)
+
+// VerifyFields contains the certficate fields to verify in the test.
+type VerifyFields struct {
+	notBefore time.Time
+	notAfter  time.Time
+	extKeyUsage x509.ExtKeyUsage
+	isCA		bool
+	org		string
```

Comment:
These violate the Go format. Please run `go fmt ./...` to have them auto-formatted.